paper/src/main/java/vg/civcraft/mc/civmodcore/CivModCoreConfig.java

```diff
@@ -17,6 +17,12 @@ public final class CivModCoreConfig extends ConfigParser {
 	private int skinCacheThreads;
 	private static final int DEFAULT_SKIN_CACHE_THREADS = Runtime.getRuntime().availableProcessors() / 2;
 
+	private boolean chunkLoadingStatistics;
+	private static final boolean DEFAULT_CHUNK_LOADING_STATISTICS = true;
+
+	private int chunkLoadingThreads;
+	private static final int DEFAULT_CHUNK_LOADING_THREADS = 1;
+
 	CivModCoreConfig(@Nonnull final CivModCorePlugin plugin) {
 		super(plugin);
```

Comment:
Annotations are cool, but don't do anything by themselves. This should be Object.requireNonNull.

Reply:
Fixed